string[] joints
string[] links
string[] sensors
string   urdf_file
geometry_msgs/Pose initial_pose
sensor_msgs/JointState initial_joint_state